.env.example

```diff
@@ -17,7 +15,7 @@ LOG_CHANNEL=stack
 # Table prefix (e.g. lychee_) of a Lychee v3 instance for migration
 DB_OLD_LYCHEE_PREFIX=
 
-# DB_CONNECTION can be sqlite, mysql or pgsql. For sqlite the other entries are
+# DB_CONNECTION can be sqlite, mysql, pgsql or sqlsrv. For sqlite the other entries are
```

Comment:
Do we want this here? We're not testing it, none of the developers use it, I'd leave it unsupported. Maybe mention it in the proper docs where there's space for a disclaimer - I already don't like how long this bit is, and I wrote it.